### ISSUE TITLE
Distribution artifacts broken?

### ISSUE THREAD

**Issue by mikesperber** [bebraw/reactabular#70]
I'm trying to use Reactabular in a ClojureScript application (so I'm not using npm), and am trying to use the artifacts in `dist` in my HTML file like so:

```
<html>
    <body>
        <div id="content"></div>
        <script src="out/goog/base.js" type="text/javascript"></script>
        <script src="reactabular.js" type="text/javascript"></script>
        <script src="../../target/table/main.js" type="text/javascript"></script>
        <script type="text/javascript">goog.require("examples.table.core");</script>
    </body>
</html>
```

That gives me:

```
TypeError: _ is undefined
```
... for this bit of code at the very beginning:

```
var _ = __webpack_require__(2);
var merge = _.merge;
            ^
```

Am I doing something wrong?

**Comment by bebraw** [comment]
Reactabular depends on lodash/underscore.

One option to get rid of that requirement would be to bundle the dependencies. That would bloat the bundle a little but it would make it easier to consume.

**Comment by mikesperber** [comment]
Thanks for the quick turnaround!

So what exactly would I need to include along with `reactabular.js`?

I've added `lodash.min.js`, but the problem persists.

**Comment by bebraw** [comment]
> So what exactly would I need to include along with reactabular.js?

Good question. The [dist build](https://github.com/bebraw/reactabular/blob/master/dist/reactabular.js#L1) contains UMD wrapper so there are actually a couple of options. It looks like it seeks lodash through `root["lodash"]` (`root` points at `window`). I suppose that's the issue.

I could change the dist build to generate `root["_"]`. Maybe that would be nice here.

Can you try these adjustments? I have a feeling I might need to tweak React entry too while at it.

**Comment by mikesperber** [comment]
Thanks, that gets me further - to "React is undefined". Now I suppose I need to figure out how to convince ClojureScript to sqeeze React between Reactabular and my code?

**Comment by bebraw** [comment]
> Thanks, that gets me further - to "React is undefined". Now I suppose I need to figure out how to convince ClojureScript to sqeeze React between Reactabular and my code?

Likely yes. There's a declaration like `root["react/addons"]`. I expect we might need to have `root["React"]` or similar for Reactabular to find React through a global.

Let me know if this does the trick. Once we get this working I'll do these changes to the distribution version. It's far easier to consume Reactabular through npm but it's definitely nice if we have a sane dist bundle out there.

**Comment by mikesperber** [comment]
`root["React"]` indeed works, thanks! Now with that in place, how do I get at the binding for `Table` itself?

**Comment by bebraw** [comment]
@mikesperber I think you should be able to see something through `Reactabular` global (the code performs `exports["Reactabular"]`).

I'll tweak `root` for the next version so you don't have to patch these by hand.

**Comment by mikesperber** [comment]
Ah, silly me. Didn't see that somehow. Thanks!

**Comment by kro12** [comment]
Same issue here. Using Underscore.js and global '_' reference getting replaced with lodash. Great component btw, simple to implement and powerful functionality out of the box. Unfortunately I will need to go with something else for now.

**Comment by bebraw** [comment]
@mikesperber @kro12 I figured out a nice fix for this. Can you have a look at [dev](https://github.com/bebraw/reactabular/tree/dev) and give it a go? See esp. [./dist](https://github.com/bebraw/reactabular/tree/dev/dist).

**Comment by mikesperber** [comment]
Yup, works. Thanks!

**Comment by bebraw** [comment]
@mikesperber Great to hear! I published `v0.7.0` with the change. I went for `0.7.0` to play it safe as I don't want to break someone's project in case there's some glitch in the dist.
